website/docs/r/virtual_network.html.markdown

```diff
@@ -56,6 +56,10 @@ resource "azurerm_virtual_network" "test" {
     environment = "Production"
   }
 }
+
+output "id_of_subnet2" {
+  value = "${azurerm_virtual_network.test.subnet.2796830261.id}"
```

Comment:
rather than using the hash value, it should be possible to access this as `${azurerm_virtual_network.test.subnet[1].id}` or `${azurerm_virtual_network.test.subnet.*.id[1]}`

Reply:
it seems do not work with set. `${azurerm_virtual_network.test.subnet[1].id}` says `Error reading config for output id_of_subnet2: parse error at 1:41: expected "}" but found "."`.

And `${azurerm_virtual_network.test.subnet.*.id[1]}` says `output.id_of_subnet2: At column 43, line 1: invalid index operation into non-indexable type: TypeString in: ${azurerm_virtual_network.test.subnet.*.id[1]}`.

Reply:
going to remove this example for the moment - once HCL2/TF 0.12 has landed this should be more accessible